src/os/bluestore/BlockDevice.cc

```diff
@@ -93,9 +93,11 @@ BlockDevice *BlockDevice::create(CephContext* cct, const string& path,
   int r = ::readlink(path.c_str(), buf, sizeof(buf) - 1);
   if (r >= 0) {
     buf[r] = '\0';
+#if defined(HAVE_SPDK)
```

Comment:
"Exclude nvme if no SPDK": 
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes. in this case, it'd be "os/bluestore:".

Reply:
@tchaikov 
Thanx, for the review, and I'll fix your request.
I've set the PR to DNM due to other issues I'm having.
ATM I'm tracking down a weird error I have where I have the decode template for  `_u32` return a value equal to the first 4 chars of the string following it.
And that is hampered by the the recent upgrade to Clang10, so I'm battling at a few front to get things going.